Sources/StreamVideoSwiftUI/Utils/Camera.swift

```diff
@@ -129,38 +129,28 @@ class Camera: NSObject, @unchecked Sendable {
             logger.error("Failed to obtain video input.")
             return
         }
-        
-        let photoOutput = AVCapturePhotoOutput()
-                        
-        captureSession.sessionPreset = AVCaptureSession.Preset.photo
+
+        captureSession.sessionPreset = AVCaptureSession.Preset.high
 
         let videoOutput = AVCaptureVideoDataOutput()
-        videoOutput.setSampleBufferDelegate(self, queue: DispatchQueue(label: "VideoDataOutputQueue"))
-  
+        videoOutput.setSampleBufferDelegate(self, queue: DispatchQueue.global(qos: .userInteractive))
```

Comment:
`setSampleBufferDelegate` says that "You must pass a serial dispatch to guarantee that audio samples will be delivered in order."

Reply:
Very good poing! :lifesaver: